src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ca/AssessmentService.java

```diff
@@ -914,4 +916,15 @@ private boolean isRequirementInAssessmentTaxon(final Requirement requirement,
 
     return match.isPresent();
   }
+
+  /**
+   * Checks the dimensionId is a valid positive integer
+   * 
+   * @param BigDecimal value
+   */
+  private void checkNegNumber(Integer value) {
+    if (Integer.signum(value) == -1) {
```

Comment:
The AC specifically states to check the value is positive, so either `signum(value) < 1` or simply `value < 1` should do it?

Reply:
updated to value < 1